<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>175</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="border">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>330</width>
     <height>216</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
    background-color: rgb(71, 71, 71);
    border: 1px solid black
}QComboBox QAbstractItemView {
    border: 1px solid black;
    background: rgb(71, 71, 71);
    selection-background-color: gray;
}QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 19px;
    border-right-width: 1px;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;}</string>
   </property>
   <property name="text">
    <string>⠀</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="pad">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>330</width>
     <height>216</height>
    </rect>
   </property>
   <property name="text">
    <string>⠀</string>
   </property>
  </widget>
  <widget class="QSlider" name="opacity">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>181</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SizeHorCursor</cursorShape>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="opacity_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Opacity</string>
   </property>
  </widget>
  <widget class="QLabel" name="config_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Configuration menu</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="spawn_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Spawn</string>
   </property>
  </widget>
  <widget class="QLabel" name="selected_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selected widget type:</string>
   </property>
  </widget>
  <zorder>pad</zorder>
  <zorder>border</zorder>
  <zorder>close_btn</zorder>
  <zorder>save_btn</zorder>
  <zorder>opacity_label</zorder>
  <zorder>opacity</zorder>
  <zorder>config_label</zorder>
  <zorder>listWidget</zorder>
  <zorder>spawn_btn</zorder>
  <zorder>selected_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
